libsolidity/ast/Types.cpp

```diff
@@ -2962,6 +2962,19 @@ MemberList::MemberMap FunctionType::nativeMembers(ContractDefinition const*) con
 			);
 		return members;
 	}
+	case Kind::DelegateCall:
+	{
+		solAssert(m_declaration, "");
+		auto const *functionDefinition = dynamic_cast<FunctionDefinition const*>(m_declaration);
+		if (functionDefinition->visibility() != Declaration::Visibility::Internal)
```

Comment:
private?

Reply:
I'll add test cases - I don't think this can hit here - I'll either also exclude it in the ``if`` or assert that it's not the case.

Reply:
Added two test cases and an assertion here.